WordPress/src/main/java/org/wordpress/android/models/ReaderBlog.java

```diff
@@ -81,6 +82,10 @@ public static ReaderBlog fromJson(JSONObject json) {
             blog.numSubscribers = json.optInt("subscribers_count");
         }
 
+        if (json.has("unseen_count")) {
+            blog.numUnseenPosts = json.optInt("unseen_count");
```

Comment:
Checking that the parameter exists with `has()` doesn't seem to be necessary here, as we're using `optInt()` which fallbacks to `0` by default.